lookout/style/format/analyzer.py

```diff
@@ -228,7 +230,8 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_request_st
             else:
                 cls.log.info("training on %d %s files", len(files), language)
             # we sort to make the features reproducible
-            X, y, _ = fe.extract_features(sorted(files, key=lambda x: x.path))
+            X, y, _ = fe.extract_features(sorted(files, key=lambda x: x.path),
```

Comment:
should probably use the standard **lang_config[relevant_section] to avoid mixing sections and to ease future adding of parameters

Reply:
Yep, figured this out in the end